python_modules/dagster/dagster/_core/instance/config.py

```diff
@@ -115,7 +116,7 @@ def dagster_instance_config(
                 dagster_config_dict["storage"],
             )
 
-    dagster_config = validate_config(schema, dagster_config_dict)
+    dagster_config = process_config(schema, dagster_config_dict)
```

Comment:
fixes a long-standing issue where default values were being ignored, will make sure callsites will not change behavior if the key was not set

Reply:
ah nice